webapp/src/views/Analysis.vue

```diff
@@ -287,21 +286,6 @@
       },
       updateAssetId () {
         this.$store.commit('updateAssetId', this.$route.params.asset_id);
-      },
```

Comment:
This "checkserveraccess" method is no longer needed imo, since we aren't relying on IP based access policies anymore.